### PR TITLE
EOS-13222: UDX masking

### PR DESCRIPTION
**Pull request by soniyamoholkar**
# Problem Statement

EOS-13222 - UDX masking

# Unit testing on RPM done

No

# Problem Description

Mask UDX APIs as well as on UI

# Solution

copied setup from given path while building
update roles and permissions while starting agent

# Test Cases
Tested on local VM
NA

### CODE REVIEW

**Comment by soniyamoholkar** [comment]
Build passed for the PR http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/cortx-manager/21/console

**Review by pawankumarsrivastava — APPROVED** [review]

File: cicd/build.sh [hunk 1]
```diff
@@ -23,6 +23,8 @@ CSM_PATH="${CORTX_PATH}csm"
 DEBUG="DEBUG"
 INFO="INFO"
 PROVISIONER_CONFIG_PATH="${CORTX_PATH}provisioner/generated_configs"
+CORTX_UNSUPPORTED_FEATURES_PATH="${CORTX_PATH}schema/unsupported_features.json"
+BRAND_UNSUPPORTED_FEATURES_PATH="/config/csm/unsupported_features.json"
```
Comment (ajaysrivas) [inline cicd/build.sh:27]: Looks like an absolute path. Is there a /config directory in our system?

File: schema/unsupported_features.json [hunk 2]
```diff
@@ -2,15 +2,15 @@
     "setup_types": [
         {
             "name":"virtual",
-            "unsupported_features": ["alerts","health", "cluster_management", "lyve_pilot"]
+            "unsupported_features": ["lyve_pilot"]
         },
         {
             "name":"5u84",
-            "unsupported_features": []
+            "unsupported_features": ["lyve_pilot"]
```
Comment (ajaysrivas) [inline schema/unsupported_features.json:9]: why "lyve_pilot" should be disabled in 5u84 and PODS by default?

File: cicd/build.sh (hunk 1 above)
Comment (soniyamoholkar) [reply cicd/build.sh:27]: -l argument will have the initial path to the repo dir for seagate-ldr. This path will be appended to that initial path to get the path to unsupported_features.json in the seagate-ldr repo dir.

File: schema/unsupported_features.json (hunk 2 above)
Comment (soniyamoholkar) [reply schema/unsupported_features.json:9]: This change has been done after discussion with @kupranay (Pranay). Initially it was not there.

File: cicd/build.sh (hunk 1 above)
Comment (ajaysrivas) [reply cicd/build.sh:27]: yes. That's why it's confusing. This should be defined as following - 

BRAND_UNSUPPORTED_FEATURES_PATH=_"config/csm/unsupported_features.json"_

It tells the code reader that it is a relative path and use it by prepending "/" whereever it's required.

File: schema/unsupported_features.json (hunk 2 above)
Comment (kupranay) [reply schema/unsupported_features.json:9]: As discussed default is lyve_pylot disabled. However on build time we are taking a decision to update config file

File: cicd/build.sh (hunk 1 above)
Comment (soniyamoholkar) [reply cicd/build.sh]: Done.

**Pull request merged**